app/views/patients/index.html.erb

```diff
@@ -1,58 +1,92 @@
 <h1>48 Hour Patient Followups</h1>
 
-<% @patients.group_by(&:registration_facility).each do |facility, patients| %>
+<div class="row">
+  <div class="col-md-6">
+    <%= bootstrap_form_with(url: patients_path, method: :get, layout: :horizontal, class: "mt-4") do |form| %>
```

Comment:
We are using this component in 2 places. We can extract this out as a partial.

Reply:
done